sys/vm/pmap.h

```diff
@@ -108,7 +108,6 @@ extern vm_offset_t kernel_vm_end;
 #define	PMAP_ENTER_WIRED	0x00000200
 #if __has_feature(capabilities)
 #define	PMAP_ENTER_NOLOADTAGS	0x00000400
-#define	PMAP_ENTER_NOSTORETAGS	0x00000800
```

Comment:
Is this flag still potentially useful for setting PTE_CRO and the vm page flag for tracking that permission?

Reply:
PTE_CRO is going to be taken from `VPO_CAPSTORE` (what was `FASTCAPDIRTY`) or the object's `NOSTORETAGS` flag; the `pmap` can look up, since it has the `vm_page_t` rather than the `vm` needing to pull out duplicate information to pass down.